system/keyboardlayouts/hungarian.xml

```diff
@@ -7,15 +7,15 @@ Default font lacks support for all characters
   <layout language="Hungarian" layout="QWERTZ">
     <keyboard>
       <row>0123456789öüó</row>
-      <row>qwertzuiopőú</row>
-      <row>asdfghjkléáű</row>
-      <row>íyxcvbnm,.-</row>
+      <row>qwertzuiopőú+</row>
+      <row>asdfghjkléáű-</row>
+      <row>íyxcvbnm,.?!@</row>
```

Comment:
not sure if we have any 'standard' regarding this, but all those characters are already available in the symbol section.

Reply:
yes there are a symbol section. I think so you can use a keyboard faster.

Reply:
Updated hungarian.xml frequently used buttons add to keyboard layout... 
fe1550f  to cd91fd8